src/main/java/API_hw2/beans/LabelNames.java

```diff
@@ -0,0 +1,200 @@
+
+package API_hw2.beans;
+
```

Comment:
Clear your beans package. You don't need all those classes, you don't need all the generated fields. Leave only necessary classes

Reply:
Beans files are generated automatically with jsonschema2pojo.  LabelNames. Limits and Prefs are used in TrelloBoard. TrelloBoard used in TrelloServiceObj

Reply:
Can you alter your JSON?

Reply:
I didn’t think about it. Thanks for this idea